hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/ClientExceptionsUtil.java

```diff
@@ -59,18 +58,23 @@ public static boolean isMetaClearingException(Throwable cur) {
     if (cur == null) {
       return true;
     }
-    return !isSpecialException(cur) || (cur instanceof RegionMovedException)
-        || cur instanceof NotServingRegionException;
+    return !regionLocationIsConsistentWithServerException(cur);
   }
 
-  public static boolean isSpecialException(Throwable cur) {
-    return (cur instanceof RegionMovedException || cur instanceof RegionOpeningException
-        || cur instanceof RegionTooBusyException || cur instanceof RpcThrottlingException
-        || cur instanceof MultiActionResultTooLarge || cur instanceof RetryImmediatelyException
-        || cur instanceof CallQueueTooBigException || cur instanceof CallDroppedException
-        || cur instanceof NotServingRegionException || cur instanceof RequestTooBigException);
+  public static boolean regionServerIsOverloadException(Throwable t) {
+    return t instanceof CallQueueTooBigException || t instanceof CallDroppedException;
   }
 
+  private static boolean regionLocationIsConsistentWithServerException(Throwable t) {
+    return t instanceof RegionOpeningException || t instanceof RegionTooBusyException
+        || t instanceof RpcThrottlingException || t instanceof RetryImmediatelyException
+        || t instanceof RequestTooBigException;
+  }
+
+  private static boolean matchExceptionWeCare(Throwable t) {
```

Comment:
I'v been recasting names so they are mbean style... 'is' methods have boolean returns...

Here I'd call it isExceptionWeCare but even this is a bad name. What does it mean 'WeCare'. What happens when this triggers? Can we get that into the name instead? Thanks.

Reply:
This function is called in client side, and the purpose of it is that if the return value of the function is true, so we are sure about how to deal with meta cache. The function name findException is too general and make it confusing to name a detailed function inside it.I tried many times and cannot find a suitable one.